.github/workflows/build.yml

```diff
@@ -66,15 +66,15 @@ jobs:
         cd ..
     - name: Initialize Dapr runtime ${{ env.DAPR_RUNTIME_VER }}
       run: |
+        sudo dapr uninstall
```

Comment:
I think you want to pass `--all` here.

Reply:
Do we still need sudo now that the binary doesn't reside in /use/local/bin?

Reply:
It shouldn't be needed if it's a clean environment.  But I'm noticing that if there was a previous dapr installation it may have set permissions in some folders that dapr init needs to write under ~/.dapr/ and some of those may not be overridable without a sudo.

Reply:
> Do we still need sudo now that the binary doesn't reside in /use/local/bin?

I had left it in case we were upgrading from an older version, but we probably don't need it going forward.